src/main/resources/xslt/outputs/odt/models.xsl

```diff
@@ -1,602 +1,603 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Less critical file, but still as a pattern, Copy/Paste commits should be avoided for all if possible.